tools/visual-pipeline-and-platform-evaluation-tool/app.py

```diff
@@ -890,6 +894,18 @@ def create_interface(title: str = "Visual Pipeline and Platform Evaluation Tool"
         elem_id="object_classification_reclassify_interval",
     )
 
+    pipeline_watermark_enabled = gr.Checkbox(
+        label="Overlay inference results on inference channels",
+        value=True,
+        elem_id="pipeline_watermark_enabled",
+    )

```

Comment:
After unchecking "Overlay inference results on inference channels," the bounding boxes no longer appear on the video from the output inference channels. 
I checked that the functionality works, but I'm wondering - what is its purpose?